geonode/base/utils.py

```diff
@@ -141,6 +142,24 @@ def build_absolute_uri(url):
     return url
 
 
+def get_resources(user):
```

Comment:
@marthamareal please either use the already available [filter](https://github.com/GeoNode/geonode/blob/cf0142a57f1fa6c59ee6d4b186d7bf8f0506bc07/geonode/base/api/permissions.py#L150) defined in ResourceBasePermissionsFilter, or make that filter use this method. Please notice that the filter_queryset does something more, for example it checks the SKIP_PERMS_FILTER settings.

Reply:
I have updated this @giohappy